scripts/prepare-restricted-environment.sh

```diff
@@ -118,6 +118,10 @@ echo "[INFO] Mirroring related images to the $my_registry registry."
 while IFS= read -r line
 do
   public_image=$(echo "${line}" | cut -d '=' -f1)
+  if [[ "$public_image" == registry.redhat.io/rhdh/* ]]; then
+    # CI Builds not public yet
+    public_image=${public_image/registry.redhat.io/quay.io}
```

Comment:
This will also change references to the rbac and postgres images, so we want to only make this change for our rhdh images:

```suggestion
    public_image=${public_image/registry.redhat.io\/rhdh/quay.io\/rhdh}
```

Reply:
But rbac and postgresql images are respectively `registry.redhat.io/openshift4/ose-kube-rbac-proxy` and `registry.redhat.io/rhel9/postgresql-15`, so we should not enter this, because of the condition above which catches only images starting with `registry.redhat.io/rhdh`.
Anyway, to be safe, I'll add what you suggest.